mscore/musescore.cpp

```diff
@@ -445,7 +445,7 @@ void MuseScore::preferencesChanged(bool fromWorkspace)
       {
       updateExternalValuesFromPreferences();
 
-      setPlayRepeats(MScore::playRepeats);
```

Comment:
Why did you decide to remove the call of all other operations included to this method, and left only the checked state setter?
Isn't the call of `setPlayRepeats` necessary here?

Reply:
It doesn't seem to be as per my testing

Reply:
@Soerboe added this code for some reason... https://github.com/musescore/MuseScore/commit/064156c267c6b3b576ac133f064644a9172f300e

Reply:
Maybe, but it didn't work, not for showing the current real status